src/routes/(console)/project-[region]-[project]/sites/site-[site]/domains/add-domain/+page.svelte

```diff
@@ -50,27 +52,41 @@
     });
 
     async function addDomain() {
+        let domain = data.domains?.domains.find((d) => d.domain === domainName);
+
+        if (!domain) {
+            try {
+                domain = await sdk.forConsole.domains.create($project.teamId, domainName);
+            } catch (error) {
+                addNotification({
+                    type: 'error',
+                    message: error.message
+                });
+
+                return;
+            }
+        }
```

Comment:
How does this behave on selfhosted?

Reply:
hmm, makes sense. we shouldn't have a redirection to verify-domains as well then, right?